prov/sm2/src/sm2_attr.c

```diff
@@ -32,129 +32,116 @@
 
 #include "sm2.h"
 
-#define SM2_TX_CAPS (OFI_TX_MSG_CAPS | FI_TAGGED )
-#define SM2_RX_CAPS (FI_SOURCE | OFI_RX_MSG_CAPS | FI_TAGGED | \
-		     FI_DIRECTED_RECV | FI_MULTI_RECV)
+#define SM2_TX_CAPS (OFI_TX_MSG_CAPS | FI_TAGGED)
```

Comment:
Clang still off

Reply:
(this was supposed to be for the line under this one...)

Reply:
Is this good enough or should we do more investigation on clang-format style?

```
#define SM2_RX_CAPS                                                   \
	(FI_SOURCE | OFI_RX_MSG_CAPS | FI_TAGGED | FI_DIRECTED_RECV | \
	 FI_MULTI_RECV)
```

Reply:
I am going to assume this is good enough, if you still don't like it... call it out